### ISSUE TITLE
UglifyJsPlugin prevents karma-webpack production `dist` work fine.  

### ISSUE THREAD

**Issue by bySabi** [webpack/karma-webpack#164]
@MikaAK @d3viant0ne when `karma-webpack` is builded with `npm run build:dist` and later use on a [example project](https://github.com/bySabi/karma--tap--boilerplate) I got this error:

 ```
 07 08 2016 11:31:09.456:ERROR [preprocess]: Can not load "webpack"!
  Error: No provider for "t"! (Resolving: preprocess -> preprocessor:webpack -> t)
    at error (/Users/myName/Dropbox/WIP/karma--tap--boilerplate/node_modules/di/lib/injector.js:22:12)
    at Object.parent.get (/Users/myName/Dropbox/WIP/karma--tap--boilerplate/node_modules/di/lib/injector.js:9:13)
```

I add `mangle: false` to UglifyJsPlugin on https://github.com/webpack/karma-webpack/blob/master/webpack.config.babel.js#L39 and new error looks better:
```
07 08 2016 11:43:02.477:ERROR [preprocess]: Can not load "webpack"!
  Error: No provider for "basePath"! (Resolving: preprocess -> preprocessor:webpack -> basePath)
    at error (/Users/myName/Dropbox/WIP/karma--tap--boilerplate/node_modules/di/lib/injector.js:22:12)
```

I wondering why not remove `UglifyJsPlugin` step?, my repo is working with current `karma-webpack` master builded with `npm run build:dev`. Is really needed webpack `optimized` on a test environment?

Please remove it and publish a new `npm` version. Many developers like me are stuck with webpack2 and karma-webpack

Thanks for the hardwork on this.

**Comment by d3viant0ne** [comment]
@bySabi - Thanks for the example repo, it should make the investigation much easier. I'll look into this today.

**Comment by bySabi** [comment]
@d3viant0ne  I update example repo package.json with my own github `karma-webpack` dist.

Current dist  file `karma-webpack.js` are builded with `npm run build:dist` and `UglifyJsPlugin` part removed.

This is my fork: https://github.com/bySabi/karma-webpack

Glad it help

**Comment by MikaAK** [comment]
#163 should solve this when it lands since we removed webpack as a build tool

**Comment by bySabi** [comment]
Solved with release `1.8.0`

👍
